testscommon/genericMocks/storerMock.go

```diff
@@ -32,6 +33,14 @@ func NewStorerMock(name string, currentEpoch uint32) *StorerMock {
 	return sm
 }
 
+func NewStorerMockWithEpoch(currentEpoch uint32) *StorerMock {
+	return NewStorerMock("Storage", currentEpoch)
+}
+
+func NewStorerMockWithDefaults() *StorerMock {
```

Comment:
We will never know what `WithDefaults` means and go back an forth with this file.
Let's clean this file and just rename this to `NewStorerMock` -> the other one is actually `NewStorerMockWithEpoch`, so another rename

Reply:
Fixed.